### ISSUE TITLE
What's the configuration of learning the mask given by `legacy/Mask_Configurations/Mymodel_10.npy`

### ISSUE THREAD

**Issue by chuan1093** [cagladbahadir/LOUPE#4]
Is it the configuration written in `scripts/train_mri.py`? 
But in the code the `nb_epochs_train` is set to just 10.  Is it enough to learn a mask with just 10 epochs?

**Comment by chuan1093** [comment]
By the way, would you please send me the trained model weights for LOUPE and individual U-Nets for different mask configurations? My e-mail is `16061093@buaa.edu.cn`.

Thanks a lot!!!

**Comment by adalca** [comment]
@chuan1093 
`nb_epoch_train` was set to 10 as an example, you will likely need more. To see whether it's enough, we recommend you track the training loss and see if it's converged.

@cagladbahadir should be able to help you with the model weights/mask

**Comment by chuan1093** [comment]
OK. Thanks a lot.
